.coveragerc

```diff
@@ -4,8 +4,20 @@ omit =
     eva/udfs/abstract_udf.py
     eva/udfs/emotion_detector.py
 
+    # Ray Execution
+    eva/executor/exchange_executor.py
+    eva/executor/ray_stage.py
+    eva/planner/exchange_plan.py
+    ray_playground.py
+    script/install/conda_eva_environment.yml
```

Comment:
Please remove the `ray_playground.py` from the pr. 
@gaurav274 Shall we remove the `conda_eva_environemnt.yml` And is this okay to exclude these files for coverage?

Reply:
We can consider adding `ray_playground.py` under the experimental directory.

Reply:
Can we do it for all the `ray` related files and fix the imports accordingly? And list experimental dir under coveragerc.

Reply:
`ray_playground.py` is super old when I investigate how to build the pipeline. I would suggest removing it, since it is not closely relevant to the current code base. 

Yes I like the idea of experimental directory.